addons/popochiu/engine/popochiu.gd

```diff
@@ -596,37 +341,22 @@ func get_text(msg: String) -> String:
 	return tr(msg) if settings.use_translations else msg
 
 
-## Gets the instance of the [PopochiuCharacter] identified with [param script_name].
+## @deprecated
+## [b]Deprecated[/b]. Now this is done by [method PopochiuICharacter.get_character_instance].
 func get_character_instance(script_name: String) -> PopochiuCharacter:
-	for rp in PopochiuResources.get_section('characters'):
-		var popochiu_character: PopochiuCharacterData = load(rp)
-		if popochiu_character.script_name == script_name:
-			return load(popochiu_character.scene).instantiate()
-	
-	printerr("[Popochiu] Character %s doesn't exists" % script_name)
-	return null
+	return C.get_character_instance(script_name)
```

Comment:
This is being very picky. And I mean VERY picky, so feel free to ignore me, but I think those methods can all be called `get_instance()` for consistency and to avoid being redundant, since they are encapsulated in their PopochiuObject interface.

So they can be `C.get_instance()`, `I.get_instance()`, `R.get_instance()`, etc.

Reply:
You're right. I'm gonna change them.

Reply:
Done.